src/Pos/Wallet/Web/Error/Util.hs

```diff
@@ -38,6 +38,6 @@ catchEndpointErrors action = catchOtherError $ tryWalletError action
         , Handler $ \(SomeException e) -> do
             usingLoggerName logName $
                 logError $ sformat ("Uncaught error in wallet method: "%shown) e
-            throwM err500
+            throwM err500 { errBody = "Internal error occured, please report this bug." }
```

Comment:
Why don't you include `e` into this message? It's not better than the previous one.